storybooknative/src/InputField/index.js

```diff
@@ -70,8 +91,8 @@ const StyledTextInput = styled(TextInput, props => {
     color: disabled ? theme.orbit.colorTextInputDisabled : theme.orbit.colorTextInput,
     fontSize: getToken(TOKENS.fontSizeInput, props),
     width: "100%",
-    height: "100%",
-    lineHeight: getToken(TOKENS.heightInput, props),
+    height: getToken(TOKENS.heightInput, props),
+    // lineHeight: getToken(TOKENS.heightInput, props), #TODO remove comment before merge
     zIndex: 2,
```

Comment:
Don't we have some zIndex values in orbit-tokens?

Reply:
We do but not for this case. These values are from the web version.

Reply:
And why does it matter that they are from the web version?

Reply:
I thought that web and RN should be as consistent as possible.

Reply:
Ok, let me rephrase, we can't we use zIndex from orbit-tokens?

I am not saying this is wrong here, I just don't understand completely why 🙂